app/send-team-notification.js

```diff
@@ -54,7 +54,12 @@ exports.sendTeamNotification = async function sendNotification(teamConfig, repor
         config.potentialAction[3].targets[0].uri = teamConfig.flairInsightsLink;
 
         config.text = thresholdTime + '<br> ![chart image](' + teamConfig.base64 + ')' + '<br><p style="color:#9B41A3;font-size:12px">' + teamConfig.compressText + "</p>";
-
+        if (teamConfig.visualizationType == "Table" || teamConfig.visualizationType == "Pivot Table") {
+            config.text = thresholdTime + '<br> ';
+        }
+        else {
+            config.text = thresholdTime + '<br> ![chart image](' + teamConfig.base64 + ')' + '<br><p style="color:#9B41A3;font-size:12px">' + teamConfig.compressText + "</p>";
```

Comment:
keep the color css in spepare file with key chart name so that i can be changed in a file rather than in code